pkg/controller/system/system_controller.go

```diff
@@ -848,8 +848,9 @@ func (r *ReconcileSystem) PrivateKeyTranmissionAllowed(client *gophercloud.Servi
 // ReconcileCertificates configures the system certificates to align with the
 // desired list of certificates.
 func (r *ReconcileSystem) ReconcileCertificates(client *gophercloud.ServiceClient, instance *starlingxv1.System, spec *starlingxv1.SystemSpec, info *v1info.SystemInfo) error {
+
```

Comment:
The blank line seems out of place. Was this added unintentionally?

Reply:
Yes, I'll remove that.